src/Microsoft.Azure.SignalR.Common/Auth/AzureADOptions.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Security.Cryptography.X509Certificates;
+
+namespace Microsoft.Azure.SignalR
+{
+    public class AzureAdOptions
+    {
+        public bool Enabled = false;
+
+        public readonly string ClientId;
+        public readonly string ClientSecret;
+        public readonly X509Certificate2 ClientCert;
+
+        public readonly string TenantId;
+
+        public string Instance = "https://login.microsoftonline.com/{0}";
+
+        public AzureAdOptions()
+        {

```

Comment:
having a default ctor with `Enabled` as false looks a bit strange

Reply:
How about we make the default ctor private and expose a static `Disabled` instance?

Reply:
```C#
        public static AzureAdOptions Disabled = new AzureAdOptions();

        private AzureAdOptions()
        {
        }
```